lib/model/CProbabilityAndInfluenceCalculator.cc

```diff
@@ -827,6 +859,42 @@ bool CProbabilityAndInfluenceCalculator::calculate(double& probability) const {
     return m_Probability.calculate(probability);
 }
 
+bool CProbabilityAndInfluenceCalculator::calculateExplainingProbabilities(
+    TStrDoubleUMap& explainingProbabilities) const {
```

Comment:
It seems wasteful copy the strings around here. This would also be fixed if you migrated to using an enum.